app/renderer/js/pages/preference/new-server-form.ts

```diff
@@ -36,21 +42,20 @@ class NewServerForm extends BaseComponent {
 		`;
 	}
 
-	init() {
+	init(): void {
 		this.initForm();
 		this.initActions();
 	}
 
-	initForm() {
+	initForm(): void {
 		this.$newServerForm = this.generateNodeFromTemplate(this.template());
-		this.$saveServerButton = this.$newServerForm.getElementsByClassName('server-save-action')[0];
+		this.$saveServerButton = this.$newServerForm.querySelectorAll('.server-save-action')[0];
 		this.props.$root.innerHTML = '';
-		this.props.$root.appendChild(this.$newServerForm);
-
-		this.$newServerUrl = this.$newServerForm.querySelectorAll('input.setting-input-value')[0];
+		this.props.$root.append(this.$newServerForm);
+		this.$newServerUrl = this.$newServerForm.querySelectorAll('input.setting-input-value')[0] as HTMLInputElement;
```

Comment:
See my comment about this in previous commit.

Reply:
This is required because querySelector returns Element type.

Reply:
OK, sounds good!